### PR TITLE
fix: main composite to lower case

### PR DESCRIPTION
**Pull request by nicoecheza**
closes https://github.com/decentraland/sdk/issues/851

### CODE REVIEW

File: packages/@dcl/inspector/src/lib/data-layer/host/fs-utils.ts [hunk 1]
```diff
@@ -41,5 +41,5 @@ export function getFileName(fileName: string, ext: string) {
 
 export function getCurrentCompositePath(sceneProvider: SceneProvider) {
   const scenePath = sceneProvider.getScene().display.title
-  return withAssetDir(`${scenePath}/main.composite`)
+  return withAssetDir(`${scenePath}/main.composite`).toLowerCase()
```
Comment (menduz) [inline packages/@dcl/inspector/src/lib/data-layer/host/fs-utils.ts:44]: don't we have a normalizePath function over there? the effect is the same but the semantic meaning would add a lot to the devEx

Comment (menduz) [reply packages/@dcl/inspector/src/lib/data-layer/host/fs-utils.ts:44]: or maybe `withAssetDir` could take care of the normalization

Comment (nicoecheza) [reply packages/@dcl/inspector/src/lib/data-layer/host/fs-utils.ts]: changed everything to use methods from `path` module instead. We want to allow paths with uppercase letters, but `.composite` files are mandatory to be lowercase

File: packages/@dcl/inspector/src/lib/logic/in-memory-storage.ts [hunk 2]
```diff
@@ -28,27 +28,40 @@ export function createInMemoryStorage(initialFs: Record<string, Buffer> = {}) {
   }
 }
 
+function normalizePath(path: string) {
+  return path.replace(/^\/+/g, '')
```
Comment (gonpombo8) [inline packages/@dcl/inspector/src/lib/logic/in-memory-storage.ts:32]: add a comment or an example about this so i dont have to understand the regexp 🙏🏼

**Review by gonpombo8 — APPROVED** [review]

**Pull request merged**